### PR TITLE
Suggested links - whitelist

### PR DESCRIPTION
**Pull request by stephanfowler**
Adds a few missing R2 nav items ('politics' etc)

### CODE REVIEW

**Comment by crifmulholland** [comment]
Hang on... lets change the football stuff while we're at it (thus removing Chad Evans from everyone's menu)

**Comment by phamann** [comment]
:+1:

**Comment by gklopper** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/onward/history.js [hunk 1]
```diff
@@ -23,27 +23,52 @@ define([
...
-    var whitelistRx = new RegExp(/^football\/[^\/]+/),
-        whitelist = [
+    var whitelist = [
+            // R2 etc
             'audioslideshows', 'business/economics', 'business/interest-rates', 'business/markets', 'business/series/andrewclarkonamerica',
             'business/series/davidgowoneurope', 'business/series/viewpointcolumn', 'business/useconomy', 'careers', 'cartoons', 'community',
             'crosswords', 'crosswords/series/cryptic', 'crosswords/series/everyman', 'crosswords/series/prize', 'crosswords/series/quick',
             'data', 'environment/climate-change', 'environment/conservation', 'environment/energy', 'environment/ethical-living',
             'environment/food', 'environment/georgemonbiot', 'environment/list/allenvironmentkeywords', 'environment/travelandtransport',
-            'environment/waste', 'football/championsleague', 'guardian-professional', 'inpictures', 'interactive', 'law', 'lifeandstyle',
```
Comment (crifmulholland) [inline static/src/javascripts/projects/common/modules/onward/history.js]: 'football/championsleague' is in there three times...

File: static/src/javascripts/projects/common/modules/onward/history.js [hunk 2]
```diff
@@ -23,27 +23,52 @@ define([
...
+            'football/scottish-league-one', 'football/scottish-league-two', 'football/euro-2016-qualifiers', 'football/friendlies', 'football/copa-america', 'football/africannationscup',
+            'football/world-cup-2018', 'football/world-cup-2014',
+
+            // football links
+            'football/competitions', 'football/fixtures', 'football/live', 'football/results', 'football/tables', 'football/teams', 'football/series/thefiver',
+            'football/series/rumourmill', 'football/transfer-window',
+
+            // NG subnav
             'artanddesign', 'australia-news', 'books', 'business/companies', 'business/stock-markets', 'cities', 'crosswords', 'education',
             'education/students', 'fashion', 'film', 'football/competitions', 'football/fixtures', 'football/live', 'football/results',
```
Comment (crifmulholland) [inline static/src/javascripts/projects/common/modules/onward/history.js]: Can remove football links from here

**Comment by crifmulholland** [comment]
Apart from duplicates :+1:

**Comment by stephanfowler** [comment]
Lotsa duplicates now removed.

**Pull request merged**